docs/modules/publisher-commandline.md

```diff
@@ -202,7 +206,7 @@ The following list of module parametrization settings can be provided either as
             MaxNodesPerDataSet=VALUE
                                       Maximum number of nodes within a DataSet/Subscription. 
                                       When more nodes than this value are configured for a 
-                                      dataSetWriter, they will be added in a separate DataSet/Subscription.
+                                      DataSetWriter, they will be added in a separate DataSet/Subscription.
```

Comment:
>DataSet/Subscription [](http://example.com/codeflow?start=86&length=20)

Is DataSet == Subscription in the OPC UA specification?

Reply:
No, in the OPC UA Spec DataSet and Subscription are totally different concepts. We combine them though in publisher.

Reply:
@cristipogacean  i think somewhere else we should describe how these both are combined by OPC Publisher. Maybe publisher.md is the place to do this.